src/promptflow/tests/sdk_cli_azure_test/e2etests/test_flow_serve.py

```diff
@@ -2,7 +2,10 @@
 
 import pytest
 
+from ..recording_utilities import is_live
 
+
+@pytest.mark.skipif(condition=not is_live(), reason="serving tests, only run in live mode.")
```

Comment:
Seems several types of tests are skipped if not live mode, why is that necessary?

Reply:
currently default mode is live but not recording, so the experience is same as before; for those skipped tests, I will open another PR to enable as much as possible in the near future.